docker-compose.yml

```
@@ -0,0 +1,36 @@
+version: '3'
+
+volumes:
+    postgres_data:
+        driver: local
+
+services:
+    # Create service demo-fake-api who user node 18 on port 3000 and code is in packages/demo-fake-api folder. Without postgress
```

```suggestion
    # Create service demo-fake-api who uses node 18 on port 3000 and code is in packages/demo-fake-api folder. Without postgres
```

And I don't understand the 'Without postgres' part

---

> And I don't understand the 'Without postgres' part

What about this?

---

me neither, I haven't read all files added by Anthony